locale/en/manageiq.po

```diff
@@ -28565,6 +28636,10 @@ msgstr ""
 msgid "Edit assignments for this Alert Profile"
 msgstr ""
 
+#: ../app/javascript/components/schedule-form/index.jsx
+msgid "Edit of \"%s\" was cancelled by the user"
```

Comment:
I know both `cancelled` and `canceled` are acceptable for 🇬🇧 and 🇺🇸 English but should they be the same here and below?

cc @akhilkr128 @MelsHyrule

Reply:
note, this can be done later... it's just something I noticed